### PR TITLE
Issue#10 chefrecs

### LINKED ISSUE #10: [Landing Page] Add horizontally scrollable Chef Cards
## User Story
As a **[Role]**, I would like ** to scroll through and see various cards with nearby Chefs on them** so that **I can explore my nearby Chef options and gather quick information that might catch my interest**.

## Detailed Description
Interspersed in the scrollable Dish cards on the Landing Page, there will be sections that are horizontally scrollable of various nearby Chef cards. Like Dish cards, the Chef cards will feature basic information about them and be clickable to take them to the Chef's landing page

## Acceptance Criteria
- [x] Given **Chef card clicked**, then **user is taken to Chef's Storefront**
- [x] Given **[Condition]**, then **[Expected Result]**
- [x] Given **[Condition]**, then **[Expected Result]**

## Tasks
- [x] Create Chef card div
- [x] Chef's profile picture
- [x] Chef's name
- [x] Placeholder for average rating
- [x] Placeholder for chef's location or text saying how far away they are
- [x] Text with Chef's cuisine/specialty
- [x] Add cards to Landing Page
- [x] Clicking on Chef cards takes user to Chef's Storefront

### PR DESCRIPTION
**Pull request by armada877**
Resolves #10 

Added ChefRec, ChefRecList, ChefScreen

ChefRec:
Props: 
  - Item
      * Id
      * name
      * image
      * rating
      * bio
      * locaiton
      * cuisine
      * onPress

ChefRecList:
Props: 
- data

Screenshot of sample ChefRecList:
IMG_0EE3D5668BED-1
From top to bottom: Name, Cuisine, Image, Bio, Location, Rating

Side-note, I think git rebase doubled all my commits, not sure how to undo that or if it's even necessary.

### CODE REVIEW

File: app/components/dish.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ const styles = StyleSheet.create({
   cardContainer: {
     alignItems: "flex-end",
     alignContent: "stretch",
-    maxHeight: "40%"
+    maxHeight: "100%"
```
Comment (shikham-8) [inline app/components/dish.js]: Could be some conflicts with changing the height since dish cards are used in different places

**Review by shikham-8 — APPROVED** [review]
Looks good to me! There might be some conflicts with changing the dish card height to 100% though since the dish cards will be used in different places.

File: app/components/dish.js (hunk 1 above)
Comment (armada877) [reply app/components/dish.js]: oh shoot i didn't even realize i changed that, i was just seeing what those did lol, i'll revert it

Comment (armada877) [reply app/components/dish.js]: So I'm not sure why, but when I reverted the change in dish.js I ended up with this:

IMG_FA0637C02C2C-1

Not sure if that's just on my end or if there's something going on?

Comment (brendon-ng) [reply app/components/dish.js]: Is there a chance we could make it horizontal like the dish cards? The dish cards are width: 100% and height: 225

**Comment by brendon-ng** [comment]
Is there a chance we could make it horizontal like the dish cards? The dish cards are width: 100% and height: 225

File: app/components/dish.js (hunk 1 above)
Comment (armada877) [reply app/components/dish.js]: for the recommendation card? so like each recommendation takes up the whole screen

Edit: moved to other thread
Edit2: idk how to reply to the other one nvm

**Comment by armada877** [comment]
> Is there a chance we could make it horizontal like the dish cards? The dish cards are width: 100% and height: 225
for the recommendation card? so like each recommendation takes up the whole screen

**Pull request merged**